server/server.js

```diff
@@ -65,22 +64,44 @@
 			 *		done(e);
 			 *  }
 			 * }*/
-			updateServer: (app, cb) => { cb(); }
+			updateServer: (app, cb) => {
+				// Sample server root set to "/" -- must align with paths throughout
+				app.use("/", express.static(rootDir, options));
+				// Open up the Finsemble Components,services, and clients
+				app.use("/Finsemble", express.static(moduleDirectory, options));
+				// For Assimilation
+				app.use("/hosted", express.static(path.join(__dirname, "..", "hosted"), options));
+
+
+				// configs/openfin/manifest-local.json and configs/other/server-environment-startup.json
+
+				// Make the config public
+				app.use("/configs", express.static("./configs", options));
+
+
+				cb();
+			}
 		};
 	// #endregion
 
 	// #region Constants
 	const app = express();
 	const rootDir = path.join(__dirname, "..", "dist");
-	const moduleDirectory = path.join(__dirname, "..", "finsemble");
+	const moduleDirectory = path.join(__dirname, "..", "Finsemble");
 	const cacheAge = 0;
 	const outputColor = chalk.yellow;
 	const PORT = process.env.PORT || 3375;
 	// #endregion
 
 	console.log(outputColor(`Server serving from ${rootDir} with caching maxAge = ${cacheAge} ms.`));
 
-	const options = { maxAge: cacheAge };
+	let options = { maxAge: cacheAge };
```

Comment:
@NinjaManatee @carterbs JPM is using our server in production. Others might as well. We should have a "production" mode which has more appropriate caching (24 hours).

We should also ensure that gzip compression is enabled.